### PR TITLE
Fix SonarCloud bugs

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Fix all current SonarCloud [bugs](https://sonarcloud.io/project/issues?id=hedera-mirror-node&resolved=false&types=BUG)
- Fix all current SonarCloud [vulnerabilities](https://sonarcloud.io/project/issues?id=hedera-mirror-node&resolved=false&types=VULNERABILITY)

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
Does not fix any code smells or security hotspots

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

**Review by xin-hedera — DISMISSED** [review]
LGTM

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/FileWatcher.java [hunk 1]
```diff
@@ -90,6 +85,8 @@ public void watch() {
 
                 valid = key.reset();
             }
+        } catch (InterruptedException e) {
+            log.info("Watch thread halted: {}", e.getMessage());
```
Comment (Nana-EC) [inline hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/FileWatcher.java:89]: Does this need a `Thread.currentThread().interrupt();` similar to the others?

Comment (steven-sheehy) [reply hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/FileWatcher.java:89]: Good point. Done.

**Review by ijungmann — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**